parachain/pallets/outbound-queue/src/api.rs

```diff
@@ -19,3 +20,12 @@ where
 	);
 	Some(proof)
 }
+
+pub fn calculate_fee<Runtime>(
+	message: Message,
+) -> Result<Fees<<Runtime as Config>::Balance>, SubmitError>
```

Comment:
Hmm, actually, should rename `Runtime` to `T`. They are the same thing really. And I think its not necessary to cast Runtime into Config.

```suggestion
pub fn calculate_fee<T>(
	message: Message,
) -> Result<Fees<T::Balance>, SubmitError>
```

Reply:
[Rename generic Runtime to T](https://github.com/Snowfork/snowbridge/pull/975/commits/5548e949c2affcb7bd70fc5c912793013346e1d5)